packages/core/src/view/AbstractGraph.ts

```
@@ -0,0 +1,1452 @@
+ * - selectionHandler: {@link SelectionHandler} for moving and cloning cells
+ *
+ * These listeners will be called in the above order if they are enabled.
+ *
+ * @category Graph
+ */
+export abstract class AbstractGraph extends EventSource {
+  container: HTMLElement;
+
+  destroyed = false;
+
+  graphModelChangeListener: Function | null = null;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)